Tests/PostgresNIOTests/New/Connection State Machine/ExtendedQueryStateMachineTests.swift

```diff
@@ -0,0 +1,15 @@
+//
+//  File.swift
+//  
+//
+//  Created by Fabian Fett on 01.02.21.
+//
+
+import XCTest
```

Comment:
Ditch Xcode template headers.
```suggestion
import XCTest
```